types/src/block/output/mod.rs

```diff
@@ -184,6 +184,48 @@ impl Output {
         }
     }
 
+    /// Returns the address that is required to unlock this [`Output`] and the alias or nft address that gets
+    /// unlocked by it, if its an alias or nft.
+    pub fn unlocked_addresses(
+        &self,
+        current_time: u32,
+        alias_transition: Option<AliasTransition>,
+        output_id: OutputId,
```

Comment:
Do you mind swapping these 2 params ? Conventionally we'd put the options at the end